CMakeLists.txt

```diff
@@ -548,63 +554,7 @@ if (ENABLE_MONGOC)
    set (PACKAGE_PREFIX "mongo-c-driver-${MONGOC_DIST_VERSION}")
    set (DIST_FILE "${PACKAGE_PREFIX}.tar.gz")
```

Comment:
```suggestion
```

I suggest grepping this file for `dist`. Several comments and lines look no longer needed.